client/keys/add.go

```diff
@@ -259,3 +259,55 @@ func SeedRequestHandler(w http.ResponseWriter, r *http.Request) {
 	seed := getSeed(algo)
 	w.Write([]byte(seed))
 }
+
+// RecoverKeyBody is recover key request REST body
+type RecoverKeyBody struct {
+	Password string `json:"password"`
+	Seed     string `json:"seed"`
+}
+
+// RecoverResuestHandler is the handler of creating seed in swagger rest server
+func RecoverResuestHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Not sure if you're refering to `Rest` or `Request` with `Resuest`, please change it accordingly